tests/unittests/test_handler/test_handler_refresh_rmc_and_interface.py

```diff
@@ -0,0 +1,56 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import cloud
+from cloudinit import distros
+from cloudinit import helpers
+from cloudinit import util
+from cloudinit import subp
+from cloudinit import netinfo
+
+from cloudinit.tests import helpers as t_help
+
+import logging
+
+LOG = logging.getLogger(__name__)
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
+    def setUp(self):
+        super(TestRsctNodeFile, self).setUp()
+        self.tmp = self.tmp_dir()
+
+    def cc_cloud(self):
+        paths = helpers.Paths({})
+        ds = None
+        return cloud.Cloud(ds, paths, {}, distros, None)
+
+    def ipv6_interface(self):
+        info = netinfo.netdev_info()
```

Comment:
It feels like these tests probably wont run on all systems.  You shuold mock netinfo.netdev_netinfo() with specific results that you want to test.

And i dont think there are actually any tests here, are there?

the 'handle' interface is a real pain to test. thats why we've mostly moved to making them very small that just call out to other functions and writing those other functions to be more easiliy testable.